src/OrbitGl/App.cpp

```diff
@@ -1651,7 +1652,7 @@ void OrbitApp::SendErrorToUi(const std::string& title, const std::string& text,
 }
 
 orbit_base::Future<ErrorMessageOr<std::filesystem::path>> OrbitApp::RetrieveModuleFromRemote(
-    const std::string& module_file_path) {
+    const std::string& module_file_path, orbit_base::StopToken* stop_token) {
```

Comment:
As you already pointed out: A pointer here is not really a good option as the caller of this function can't really promise that the object exists longer than the function call.

Also: I would prefer to have the stop_token as the first parameter as that's what std::jthread is expecting as well and we might be able to standardize that in the future.